crates/re_viewer/src/misc/transform_cache.rs

```diff
@@ -186,7 +196,9 @@ impl TransformCache {
                     continue;
                 }
                 Ok(None) => reference_from_entity,
-                Ok(Some(child_from_parent)) => reference_from_entity * child_from_parent,
+                Ok(Some(child_from_parent)) => {
+                    transform_affine3(reference_from_entity, child_from_parent)
+                }
```

Comment:
I don't get it - why not 

```suggestion
                Ok(Some(child_from_parent)) => reference_from_entity * child_from_parent,
```